dvc/repo/live.py

```diff
@@ -15,10 +17,10 @@ def create_summary(out):
 
     metrics, plots = out.repo.live.show(str(out.path_info))
 
-    html_path = out.path_info.with_suffix(".html")
+    html_path = out.path_info.fspath + "_dvc_plots"
```

Comment:
Is there a reason for this change? 

Should we update on DVCLive that `{path}.html` is no longer the default html output?

Reply:
Good catch! To support images, the image files must be stored alongside the html, so the output is now a directory instead of a single file.

I would probably opt to do something like `dvcvlive/dvc_plots` rather than append them with underscores. Any other ideas are welcome.

Reply:
I have no strong preference but if we go with `dvcvlive/dvc_plots`, the `.html` output will now be handled like the `.tsv` plots (i.e. based on `cache` option of the output). 

Previously, the `.html` was not tracked by DVC and it was up to the user whether to track it with Git/DVC or just ignore.

Reply:
Ah, that's probably why @pared did it this way. Maybe just `dvclive_plots` then?

Another option would be to reorganize dvclive outputs so that the tsv files are under a subdirectory like `dvclive/logs` or `dvclive/tsv` and the html is another subdirectory like `dvclive/plots` or `dvclive/html`.

Reply:
I kind of like `dvclive/tsv` & `dvclive/html`. 

Honestly, the word `plots` for referring to the HTML sounds kind of confusing to me because the `.tsv` files are also `dvc plots` in a sense.

Moving the outputs to subdirectories would require some changes in DVC core but it feels like a good direction. I might consider also moving the summary `.json`  to `dvclive/summary.json`.

Reply:
Maybe we can continue the discussion in https://github.com/iterative/dvclive/issues/148 as this is a comment in a closed P.R.